packages/contracts-bedrock/test/Specs.t.sol

```diff
@@ -438,7 +585,7 @@ contract Specification_Test is CommonTest {
     }
 
     /// @dev Adds a spec for a function.
-    function _addSpec(string memory _name, bytes4 _sel, bool _auth, bool _pausable) internal {
+    function _addSpec(string memory _name, bytes4 _sel, Role _auth, bool _pausable) internal {
```

Comment:
I am confused about what the enum does here in practice

Reply:
It specifies what is allowed to access a particular function. For example, there's a `CHALLENGER` role that's used to specify that it can delete l2 outputs.

Reply:
Basically it's documenting msg.sender requirement used in `require(msg.sender == x)`. The Role is a concept for that check, rather than a specific address.